app/models/organization.rb

```diff
@@ -13,7 +13,7 @@ class Organization < ActiveRecord::Base
   attr_accessor :is_public_submission
 
   validates_presence_of :name
-  validates_presence_of :github_org, if: :is_public_submission
+  validates_presence_of :github_org
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.